changelog.d/1988-change.rst

```diff
@@ -0,0 +1,2 @@
+Deprecated the ``bdist_rom`` command. Binary packages should be built as wheels instead.
```

Comment:
```suggestion
Deprecated the ``bdist_rpm`` command. Binary packages should be built as wheels instead.
```

Reply:
Thanks! Mistyped that nearly every time but missed one :)